automl/README.rst

```diff
@@ -38,7 +38,7 @@ In order to use this library, you first need to go through the following steps:
 .. _Select or create a Cloud Platform project.: https://console.cloud.google.com/project
 .. _Enable billing for your project.: https://cloud.google.com/billing/docs/how-to/modify-project#enable_billing_for_a_project
 .. _Enable the Cloud AutoML API.:  https://cloud.google.com/automl
-.. _Setup Authentication.: https://googleapis.github.io/google-cloud-python/latest/core/auth.html
+.. _Setup Authentication.: https://googleapis.dev/python/google-api-core/latest/auth.html
```

Comment:
Why the `google-` prefix for `api_core`, but not for `automl`, etc.?

Reply:
I used the dns name for libraries that correspond to products and the distribution name if the package didn't correspond to an Cloud API. The goal is to eventually have consistent urls across languages for APIs googleapis.dev/{language}/automl/latest